### ISSUE TITLE
Link to Capacity Planning Document

### ISSUE THREAD

**Issue by jrjenk** [MicrosoftDocs/ATADocs#468]
tl;dr:
In Next Steps change the link with text Defender for Identity sizing tool from https://aka.ms/trisizingtool to https://docs.microsoft.com/en-us/defender-for-identity/capacity-planning

Reason:

Your link in the Next steps section for the Defender for Identity Sizing tool leads to what I believe is an outdated aka.ms link.

I would consider instead linking to the docs.microsoft.com page that talks about the sizing tool and then has the current link to download the sizing tool, which is https://docs.microsoft.com/en-us/defender-for-identity/capacity-planning

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d74f052e-fd05-b96e-fb2a-0f67b7a6d4da
* Version Independent ID: 80591357-4e2e-be6c-e31f-c92b2d5215b0
* Content: [Microsoft Defender for Identity architecture](https://docs.microsoft.com/en-us/defender-for-identity/architecture)
* Content Source: [ATPDocs/architecture.md](https://github.com/Microsoft/ATADocs/blob/master/ATPDocs/architecture.md)
* Service: **microsoft-defender-for-identity**
* GitHub Login: @dcurwin
* Microsoft Alias: **dacurwin**

**Comment by dcurwin** [comment]
@jrjenk  - Thanks for the comment! We've updated the document.
